src/libs/AccountUtils.ts

```diff
@@ -8,4 +8,12 @@ const isValidateCodeFormSubmitting = (account: OnyxEntry<Account>) =>
 /** Whether the accound ID is an odd number, useful for A/B testing. */
 const isAccountIDOddNumber = (accountID: number) => accountID % 2 === 1;
 
-export default {isValidateCodeFormSubmitting, isAccountIDOddNumber};
+function isDelegateOnlySubmitter(account: OnyxEntry<Account>): boolean {
+    const [delegate] = account?.delegatedAccess?.delegates ?? [];
+    if (!delegate) {
+        return false;
+    }
+    return delegate?.role === CONST.DELEGATE_ROLE.SUBMITTER;
```

Comment:
Will this work if there is more than one delegate? That is, if the delegator has multiple delegates, we need to make sure we're checking if the correct one is a submitter. We should look for the email match